core/node/proof_data_handler/src/tee_request_processor.rs

```diff
@@ -2,20 +2,19 @@ use std::sync::Arc;
 
 use axum::{extract::Path, Json};
 use zksync_config::configs::ProofDataHandlerConfig;
-use zksync_dal::{tee_proof_generation_dal::TeeType, ConnectionPool, Core, CoreDal};
+use zksync_dal::{ConnectionPool, Core, CoreDal};
 use zksync_object_store::ObjectStore;
-use zksync_prover_interface::api::{
-    GenericProofGenerationDataResponse, RegisterTeeAttestationRequest,
-    RegisterTeeAttestationResponse, SubmitProofResponse, SubmitTeeProofRequest,
-    TeeProofGenerationDataRequest,
+use zksync_prover_interface::{
+    api::{
+        RegisterTeeAttestationRequest, RegisterTeeAttestationResponse, SubmitProofResponse,
+        SubmitTeeProofRequest, TeeProofGenerationDataRequest, TeeProofGenerationDataResponse,
+    },
+    inputs::TeeVerifierInput,
 };
-use zksync_tee_verifier::TeeVerifierInput;
```

Comment:
You did a good job moving the input from this crate -- the purpose of `zksync_*_interface` crates is to detach different "parts" of communication, and it serves the purpose, e.g. API can serve and receive data not knowing how it's actually used (because it does not care).
This crate can now be removed from the dependencies.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2333/commits/84238811c3aaefac3ed17208df202955b827ff4e#diff-5948891da9d6c6726f73866f6f4854fd3a059756442499105403090d41e221ddL26). ✅